CONTRIBUTING.md

```diff
@@ -1,20 +1,24 @@
 # How to Contribute 🛠️
 
-If you want to contribute to the 3Box project you've come to the right place! We build very openly so that we can extend the direction of the project to the community over time. We're excited to have you here building with us. 🎉
+This document provides an overview of how you can contribute to 3Box. We're excited to have you building with us. 🎉
 
 ## Getting Started
+This section contains useful links to get you started building.
 
-### View the 3Box Project Board
+### View the Project Board
 
-To view our active work, [view the 3Box project board](https://app.zenhub.com/workspace/o/uport-project/3box/boards?repos=139279908,146687169,146688532,145592766,145133349)
-or [download the Zenhub browser extension](https://www.zenhub.com/extension).
+To view our active work, [view the 3Box project board](https://app.zenhub.com/workspace/o/uport-project/3box/boards?repos=139279908,146687169,146688532,145592766,145133349) or [download the Zenhub browser extension](https://www.zenhub.com/extension).
 
+### Join the Chat
 
-### Join the 3Box Chat
+[Join the 3Box Community](https://mailchi.mp/c671ca2b8093/3box) so you can connect with other developers and the core team on our Discord Chat.
 
-[**Join the 3Box Community**](https://mailchi.mp/c671ca2b8093/3box) so you can connect with other developers and the core team on our Discord Chat.
+### Report a Bug
+
+[Copy our Bug Template](./.github/ISSUE_TEMPLATE/bug_report.md) and create a new issue in the appropriate repo using the template.
```

Comment:
I think we can just link to creating new issues in:
* 3box-js
* 3box-activity
* 3box-hash-server
* 3box-dapp

Just links like this: https://github.com/uport-project/3box-js/issues/new